openmcyclus/depletion.py

```diff
@@ -95,86 +87,70 @@ def update_materials(self, comp_list, materials):
 
         return material_ids, mats
 
-    def read_microxs(self, path):
-        '''
-        Reads .csv file with microscopic cross sections. The
-        csv file is assumed to be named "micro_xs.csv". This will need to
-        be relative to Cyclus input file location
-
-        Parameters:
-        -----------
-        path: str
-            path of directory holding the files for/from OpenMC        
-
-        Returns:
-        --------
-        microxs: object
-            microscopic cross section data
-        '''
-        microxs = od.MicroXS.from_csv(str(path + "micro_xs.csv"))
-        return microxs
-
-    def run_depletion(self, path, comps):
+    def run_depletion(self, flux, materials, microxs):
         '''
         Run the IndependentOperator class in OpenMC to perform
-        transport-independent depletion. This method is only 
-        used in the test suite, and not in the OpenMCyclus 
+        transport-independent depletion. This method is only
+        used in the test suite, and not in the OpenMCyclus
         archetype
```

Comment:
Rather than have this function in the source code, I think it makes more sense to put it in the test suite since that is its purpose. It seems that this function uses class attributes, all of which you could pass into the function as parameters.